src/openforms/formio/api/validators.py

```diff
@@ -96,9 +96,7 @@ def __call__(self, value: UploadedFile) -> None:
                 return
 
             raise serializers.ValidationError(
-                _("The file '{filename}' is not a {file_type}.").format(
-                    filename=value.name, file_type=f".{ext}"
-                )
+                _("The provided file is not a {file_type}.").format(file_type=f".{ext}")
```

Comment:
I think there is also an occurrence on line 76 that uses the filename in the error?